src/NUnitTestExtractor/NUnitTestExtractorTests/TestExtractorTests.cs

```diff
@@ -205,6 +205,40 @@ public static void ParsingLevel_FunctionWrittenInDifferentCases_WillParseWithout
         {
             Assert.AreEqual((NUnitTestExtractorApp.Level.Function), NUnitTestExtractorApp.ParseLevel(level));
```

Comment:
You should also add a ParsingLevel_TestcaseWrittenInDifferentCases_WillParseWithoutError test like the other 3 above.